.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+* @abhinavrastogi-harness @knagurski
+
+/packages/* @abhinavrastogi-harness @knagurski
```

Comment:
i think this is too generic as well.. let's add owners for specific packages.
we can add you, me and Kevin for ui and views.
@srdjan-harness should be the owner for forms, pipeline-graph and yaml-editor.
@praneshg239 should be the owner for core-design-system

Reply:
Done.